vogon/urls.py

```diff
@@ -52,8 +52,10 @@
     url(r'^accounts/profile/', views.dashboard, name='dashboard'),
     url(r'^accounts/settings/$', views.user_settings),
     url(r'^accounts/register/$', views.register),
+    url(r'^activity/$', views.recent_activity),
     url(r'^accounts/logout/$', 'django.contrib.auth.views.logout',
                           {'next_page': '/accounts/login/'}),
+    url(r'^activity/$', views.recent_activity),
```

Comment:
@erickpeirson : We have added that for testing purposes. we would have the recent activity on our homepage